rotkehlchen/chain/binance_sc/modules/zerox/decoder.py

```diff
@@ -0,0 +1,29 @@
+from typing import TYPE_CHECKING
+
+from rotkehlchen.chain.binance_sc.modules.zerox.constants import SETTLER_ROUTERS
+from rotkehlchen.chain.ethereum.modules.zerox.constants import ZEROX_ROUTER
+from rotkehlchen.chain.evm.decoding.zerox.decoder import ZeroxCommonDecoder
+from rotkehlchen.chain.polygon_pos.modules.zerox.constants import ZEROX_FLASH_WALLET
+
+if TYPE_CHECKING:
+    from rotkehlchen.chain.base.node_inquirer import BaseInquirer
+    from rotkehlchen.chain.evm.decoding.base import BaseDecoderTools
+    from rotkehlchen.user_messages import MessagesAggregator
+
+
+class ZeroxDecoder(ZeroxCommonDecoder):
+
+    def __init__(
+            self,
+            evm_inquirer: 'BaseInquirer',
```

Comment:
use `from rotkehlchen.chain.binance_sc.node_inquirer import BinanceSCInquirer` instead of this one. Equivalent for the other chains if they use the evmInquirer in zerox